bedrock/products/templates/products/vpn/pricing.html

```diff
@@ -37,7 +37,7 @@
 {% endblock %}
 
 {% block sub_navigation %}
-  {% include 'products/vpn/includes/subnav.html' %}
+  {% include 'products/vpn/includes/subnav-refresh.html' %}
 {% endblock %}
```

Comment:
This one also {% extends "products/vpn/base.html" %}, so the same case?

https://github.com/mozilla/bedrock/blob/178aa6b762bff6b2741d5acd1e89f5d0e64ff1a7/bedrock/products/templates/products/vpn/base.html#L16-L18

```suggestion
```